js/danoni_main.js

```
@@ -4838,74 +4866,118 @@ const titleInit = () => {
+	if (g_headerObj.musicSelectUse) {
+		if (getQueryParamVal(`scoreId`) !== null) {
+			g_headerObj.viewLists = [];
+			g_headerObj.musicNos.forEach((val, j) => {
+				if (val === g_settings.musicIdxNum) {
+					g_headerObj.viewLists.push(j);
+					tmpCreatorList.push(g_headerObj.creatorNames[j]);
+				}
+			});
+			divRoot.appendChild(drawBackArrow(g_headerObj.viewLists[0] + 1));
+			loadLocalStorage(g_settings.musicIdxNum);
+			makeInfoWindow(g_msgInfoObj.W_0041.split(`{0}`).join(g_localStorageUrl), ``,
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)